webapp-django/crashstats/settings/base.py

```diff
@@ -588,21 +588,8 @@ def filter(self, record):
             "access_key": config("resource.boto.access_key", None),
             "bucket_name": config("resource.boto.bucket_name", "crashstats"),
             "region": config("resource.boto.region", "us-west-2"),
-            "prefix": config("resource.boto.prefix", ""),
-            # NOTE(willkg): In the local dev environment, we need to use a
-            # HostPortS3ConnectionContext which requires these additional configuration bits. The
-            # defaults are taken from the config sections of the relevant classes.
-            "resource_class": config(
-                "resource.boto.resource_class",
-                "socorro.external.boto.connection_context.RegionalS3ConnectionContext",
-            ),
-            "host": config("resource.boto.host", None),
-            "port": config("resource.boto.port", None),
-            "secure": config("resource.boto.secure", True),
-            "calling_format": config(
-                "resource.boto.calling_format",
-                "boto.s3.connection.OrdinaryCallingFormat",
-            ),
```

Comment:
We don't use host, port, or secure anymore. Those are replaced with endpoint_url and only used in the dev environment.

We never used prefix, so I nixed that.

The resource_class won't change anymore, so I hardcoded that.

We no longer need calling_format.

None of these changes should affect server environments.